apps/pie-microsite/src/_11ty/shortcodes/tokensTable/index.js

```diff
@@ -65,6 +66,47 @@ const buildTokenExampleElement = (token, tokenType, tokenMetadata) => {
     return tokenExampleElementHandler[tokenType](token, tokenMetadata);
 };
 
+/**
+ * Builds the global token used element displayed primarily by alias tokens
+ * @param {string} globalToken the global token referenced by this alias token
+ * @returns a <span> HTML string containing the global token used
+ */
+const buildGlobalTokenUsedElement = globalToken => {
+    const globalTokenUsedElement = `
+    <span class="c-tokensTable-tokenDescription">
+      <span class="u-font-bold u-showAboveWide">Global token used:</span> 
+      <span class="c-tokensTable-token c-tokensTable-token--light">${globalToken}</span>
+    </span>`;
+
+    return deindentHTML(globalTokenUsedElement);
+};
+
+/**
+ * Builds the overall token description element for each type of token. The description content differs based on the type of token.
+ * @param {string} token the token value i.e. #000, #ffffff, #000|0.85 or #000000|0.85
+ * @param {*} tokenType the type of token i.e. color, spacing, radius
+ * @param {*} tokenMetadata the metadata for the token. data such as descriptions
+ * @returns {string} - the description HTML string
+ */
+const buildTokenDescriptionElement = (token, tokenType, tokenMetadata) => {
+    const tokenDescriptionElementHandler = {
+        [tokenTypes.COLOR]: buildColorDescription,
+        default: () => (tokenMetadata.description
+            ? `<span class="c-tokensTable-tokenDescription">${tokenMetadata.description}</span>`
+            : '')
+    };
+
+    let description = tokenDescriptionElementHandler[tokenType]
```

Comment:
What do you think of this syntax? I think it's a little more descriptive
```suggestion
    let description = tokenType in tokenDescriptionElementHandler
```

Reply:
<img width="660" alt="Screenshot 2023-01-27 at 09 58 51" src="https://user-images.githubusercontent.com/16766185/215058739-16a2bf5e-37dd-4336-8c65-192a8cf85b25.png">

I don't mind - but trying this locally in a sandbox doesn't work - am I doing this right?

Reply:
ah, but that returns a boolean whereas we want a function output